File: python/MigratoryBirds.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the migratoryBirds function below.
def migratoryBirds(arr):
    bird_count = [0] * 5
    for i in arr:
        bird_count[i-1] += 1
    print (bird_count)
    return (bird_count.index(max(bird_count))+1)

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    arr_count = int(input().strip())

    arr = list(map(int, input().rstrip().split()))

    result = migratoryBirds(arr)

    fptr.write(str(result) + '\n')

    fptr.close()

File: scala/MigratoryBirds.scala
import java.io._
import java.math._
import java.security._
import java.text._
import java.util._
import java.util.concurrent._
import java.util.function._
import java.util.regex._
import java.util.stream._
import scala.collection.immutable._
import scala.collection.mutable._
import scala.collection.concurrent._
import scala.concurrent._
import scala.io._
import scala.math._
import scala.sys._
import scala.util.matching._
import scala.reflect._

object Solution {

    // Complete the migratoryBirds function below.
    def migratoryBirds(arr: Array[Int]): Int = {
        var bird_count = ArrayBuffer(0,0,0,0,0)
        for (i <- arr)
            bird_count(i-1) += 1
            
        //return (bird_count.index(max(bird_count))+1)
        return (bird_count.indexOf(bird_count.max) + 1)

    }

    def main(args: Array[String]) {
        val printWriter = new PrintWriter(sys.env("OUTPUT_PATH"))

        val arrCount = StdIn.readLine.trim.toInt

        val arr = StdIn.readLine.replaceAll("\\s+$", "").split(" ").map(_.trim.toInt)
        val result = migratoryBirds(arr)

        printWriter.println(result)

        printWriter.close()
    }
}
